ngraph/core/include/openvino/op/result.hpp

```diff
@@ -19,25 +19,19 @@ class OPENVINO_API Result : public Op {
     /// \brief Allows a value to be used as a function result.
     ///
     /// \param arg Node that produces the input tensor.
-    Result(const Output<Node>& arg, bool needs_default_layout = false);
+    Result(const Output<Node>& arg);
+
+    OPENVINO_DEPRECATED("This constructor is redundant, use Result(const Output<Node>& arg) instead.")
+    Result(const Output<Node>& arg, bool);
 
     bool visit_attributes(AttributeVisitor& visitor) override;
     void validate_and_infer_types() override;
 
     std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& new_args) const override;
 
-    void set_needs_default_layout(bool val) {
-        m_needs_default_layout = val;
-    }
-    bool needs_default_layout() const {
-        return m_needs_default_layout;
-    }
```

Comment:
I think this methods also needs to be deprecated first before we delete them.

Reply:
Done.